bauhaus/main.py

```diff
@@ -30,7 +41,7 @@ def parseArgs():
     parser.add_argument(
         "--conditionTable", "-t",
         action="store", metavar="CONDITION_TABLE.CSV",
-        required=True,
+        #required=True,
```

Comment:
It was a by produce of the attempt to start a `help()` feature. In order to run `help`, you shouldn't be required to give a condition table argument.  I agree this is shaky.